aw_datastore/storages/sqlite.py

```diff
@@ -0,0 +1,236 @@
+from typing import Optional, List
+from datetime import datetime, timezone, timedelta
+import json
+import os
+import logging
+
+import sqlite3
+
+from aw_core.models import Event
+from aw_core.dirs import get_data_dir
+
+from .abstract import AbstractStorage
+
+logger = logging.getLogger(__name__)
+
+LATEST_VERSION=1
+
+# The max integer value in SQLite is signed 8 Bytes / 64 bits
+MAX_TIMESTAMP = 2**63-1
+
+CREATE_BUCKETS_TABLE = """
+    CREATE TABLE IF NOT EXISTS buckets (
+        id TEXT PRIMARY KEY,
+        name TEXT,
+        type TEXT NOT NULL,
+        client TEXT NOT NULL,
+        hostname TEXT NOT NULL,
+        created TEXT NOT NULL
+    )
+"""
+
+CREATE_EVENTS_TABLE = """
+    CREATE TABLE IF NOT EXISTS events (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        bucket TEXT NOT NULL,
```

Comment:
This is still `TEXT` though

Reply:
Doesn't seem like the change was made despite you checking the box above (I unchecked it)

Reply:
Ah, good point. Fixed